javascript.js

```diff
@@ -0,0 +1,27 @@
+const myOpen = document.querySelector('.burger');
+const nav = document.querySelector('.header-menu');
+const myClose = document.querySelector('.close');
+const myLinks = document.querySelector('.links');
+
+myOpen.onclick = () => {
+  nav.classList.add('open');
+  myOpen.classList.add('display');
+};
+myClose.onclick = () => {
+  this.parentElement.classList.remove('open');
+  myOpen.classList.remove('display');
+};
+myLinks.onclick = () => {
+  this.parentElement.classList.remove('open');
+  myOpen.classList.remove('display');
+};
+const MidLink = document.getElementById('about');
+MidLink.addEventListener('click', () => {
+  this.parentElement.classList.remove('open');
```

Comment:
You did great work displaying the correct part of the page 👍🏽 but same as above, seems like `parentElement` here is undefined, which is making it impossible to get the `classList` and edit it to make the menu disappear. And as required 🤓  when the user clicks (or taps) the close (❌ ) button, the mobile menu should disappear. 😶‍🌫️